src/utils/createI13nNode.js

```diff
@@ -15,6 +15,9 @@ var hoistNonReactStatics = require('hoist-non-react-statics');
  * @method createI13nNode
  */
 module.exports = function createI13nNode (Component, options) {
+    if (!Component) {
+        return;
+    }
```

Comment:
We should print a warning message in dev env to help with debugging, inside of hiding the issue.